meteor-app/imports/ui/layouts/ordering.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { Sidebar, Menu, Icon } from 'semantic-ui-react'
+import { Switch, Route } from 'react-router-dom'
+import NotFound from '/imports/ui/not-found'
+import OrderingMainContainer from '/imports/ui/ordering/ordering-main-container'
```

Comment:
Please refer to the section in nuclino on file naming standards. `container` is not to be used. Main doesn't really provide any information. Please have another go at this